### PR TITLE
qubes-fwupd - system firmware update methods

### PR DESCRIPTION
**Pull request by Asiderr**
Signed-off-by: Norbert Kamiński <norbert.kaminski@3mdeb.com>

### CODE REVIEW

File: blog/content/post/2020-09-17-qubes_fwupd_heads_uefi.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+tags:
+  - qubes
+  - fwupd
+  - firmware
+  - security
+  - os-dev
+  - heads
+  - UEFI
+categories:
```
Comment (artur-rs) [inline blog/content/post/2020-09-17-qubes_fwupd_heads_uefi.md:22]: @Asiderr drop unused categories

File: blog/content/post/2020-09-17-qubes_fwupd_heads_uefi.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
+---
+title: Reasonably secure way to update your system firmware
+abstract: 'As you may know from the previous blog post, the qubes-fwupd is the
+          wrapper that allows you to update the firmware of your devices in the
+          Qubes OS. This time I will briefly describe the new features, whereby
+          you will securely update your system firmware.'
+cover: /covers/qubes&3mdeb_logo.png
+author: norbert.kaminski
+layout: post
+published: false
```
[the remark this answers is not in the document]
Comment (Asiderr) [reply blog/content/post/2020-09-17-qubes_fwupd_heads_uefi.md:10]: @artur-rs I've used the script, but I've changed published var manually.

File: blog/content/post/2020-09-17-qubes_fwupd_heads_uefi.md (hunk 1 above)
Comment (Asiderr) [reply blog/content/post/2020-09-17-qubes_fwupd_heads_uefi.md:22]: @artur-rs Fixed

File: blog/content/post/2020-09-17-qubes_fwupd_heads_uefi.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (artur-rs) [reply blog/content/post/2020-09-17-qubes_fwupd_heads_uefi.md]: @Asiderr there is no need for that

**Review by artur-rs — APPROVED** [review]

**Pull request merged**